src/book/02-system/01-foundations/02-build-system.mdx

```diff
@@ -344,6 +345,44 @@ The `nuc7i7bnh` target should be used for building and running code on the robot
 
 The `target` command will download (or, if needed, build) the Docker image for the specified target and then mark that target as active.
 
+##### Tests
+
+The `tests` command allows you to run built unit tests. It will automatically run tests in parallel, and dump a timestamped log file to the project directory.
+Currently, we use `CTest` which is included as part of `CMake`.
+
+In order to run unit tests, they must first be built, this can be done with
+
+```bash
+./b configure -- -DBUILD_TESTS=ON
+./b build
+```
+
+The `tests` command provides the following subcommands
+
+| Subcommand | Description                 |
+| ---------- | --------------------------- |
+| list       | List all built tests        |
+| run        | Run all or individual tests |
+
+If you would like to run a individual test (or group of tests), you can pass the `run` subcommand a string and it will only run tests that match the given string.
+Alternatively, `run` will run all built tests if no string is passed.
+
+For example, if you would like to run all tests with the string 'vision' in their name, you would do
+
+```bash
+./b tests run vision
+```
+
+The `run` subcommand also accepts the following flags
+
+| Flag                               | Description                                               |
+| ---------------------------------- | --------------------------------------------------------- |
+| -V or --verbose                    | Enable verbose output from tests                          |
+| -VV or --extra-verbose             | Enable more verbose output from tests                     |
+| -Q or --quiet                      | Make ctest not print to stdout                            |
```

Comment:
```suggestion
| `-Q` or `--quiet`                      | Make CTest not print to stdout                            |
```

Reply:
The CMake documentation uses lower case for ctest, but you already have "CTest" on the rest of this page.